packages/x-adapter-platform/src/schemas/response/models/facets/types.ts

```diff
@@ -0,0 +1,12 @@
+import { FacetModelName } from '@empathyco/x-types';
+import { Schema } from '@empathyco/x-adapter-next';
+
+/**
+ * Facet configuration containing the model name and the schema.
+ */
+export type FacetConfig = { modelName: FacetModelName; getSchema: () => Schema };
+
+/**
+ * Dictionary grouping facets configurations.
+ */
+export type FacetsConfig = Record<string, FacetConfig>;
```

Comment:
I would suggest to set the property that needs to be the key of the record:

```suggestion
export type FacetsConfig = Record<Facet['id'], FacetConfig>;
```

Reply:
Sure, thanks! I've changed it.